src/solver/utils/ortools_utils.cpp

```diff
@@ -227,9 +227,29 @@ bool solveAndManageStatus(MPSolver* solver, int& resultStatus, const MPSolverPar
     return resultStatus == OUI_SPX;
 }
 
+static bool doWeGiveBasisToSolver(const SingleOptimOptions& options,
+                                  const MPSolver* solver,
+                                  const Antares::Optimization::PROBLEME_SIMPLEXE_NOMME* Probleme)
+{
+    if (solverSupportsWarmStart(solver->ProblemType()) && Probleme->basisExists()
```

Comment:
Why not just `return solverSupportsWarmStart(solver->ProblemType()) && Probleme->basisExists() && options.solverAcquiresBasis`